### PR TITLE
Use pykg2tbl inside Broker

### PR DESCRIPTION
**Pull request by JotaFan**
Replace SparqlWrapper for pykg2tbl.
Adding some testing on the changed functions.

dependent on:
https://github.com/envri-fair/marine-eov-broker/pull/20

### CODE REVIEW

File: src/marine_eov_broker/MarineRiBroker.py [hunk 1]
```diff
@@ -138,61 +138,68 @@ def build_vocabularies(self, eov) -> dict:
...
+        Query vocabulary server and search for P01 parameters names corresponding
+            to a A05 EOV term.
+
         Arguments :
         eov: variable string
         """
-        sparql = SPARQLWrapper(self.vocabularies_server)
+        kgsource = KGSource.build(self.vocabularies_server)
+        query_result = kgsource.query(DEFAULT_QUERY_STRINGS[eov])
+        return query_result._data
```
Comment (marc-portier) [inline src/marine_eov_broker/MarineRiBroker.py]: `_data` ?? points to a guaranteed internal dict build from either rdflib or sqlwrapper source I presume?

however ties in with the dict based `QueryResult` implementation?
maybe cleaner to have/use some `to_dict()` on the `QueryResult`?

File: src/marine_eov_broker/MarineRiBroker.py [hunk 2]
```diff
@@ -394,51 +401,59 @@ def submit_request(self,
...
+        linked_var,
+        linked_dataset,
+    ) -> list:
+        # Build Knowledge source from endpoint
+        #   (could be a list of endpoint, files)
+        kgsource = KGSource.build(endpoint)
+        # Query response as a QueryResult class
+        query_result = kgsource.query(query)
+        # Get the contents of the linked_var as a list
+        content_var = query_result.to_dict()[linked_var]
```
Comment (marc-portier) [inline src/marine_eov_broker/MarineRiBroker.py:461]: this is better --> see previous comment 
pls use `query_result.to_dict()` and not `query_result._data` to reduce dependency-surface

File: src/marine_eov_broker/MarineRiBroker.py [hunk 3]
```diff
@@ -394,51 +401,59 @@ def submit_request(self,
...
-                             query_max_lon,
-                             query_max_lat,
-                             output_format,
-                             linked_var,
-                             linked_dataset,) -> list:
-        wrapper = SPARQLWrapper(endpoint)
-        wrapper.setReturnFormat(JSON)
-        wrapper.setQuery(query)
-        results = wrapper.queryAndConvert()
+    def submit_sparql_query(
```
Comment (marc-portier) [inline src/marine_eov_broker/MarineRiBroker.py:440]: so after all no `submit_sparql_named_query` variant that also abuses the templating 
(further cleaning the ipynb code not worth it?)

File: src/marine_eov_broker/NVSQueries.py [hunk 4]
```diff
@@ -1,107 +1,25 @@
...
-  ?dt ?rel  ?v .  filter(regex(str(?v),'S07/current/'))
-   }
-  
-     ?dt ?rel1 ?v1 .
-       ?dt ?rel2 ?v2 .
-        ?dt ?rel3 ?v3 .
-   optional {    ?dt ?rel33 ?v33 . filter(regex(str(?v33),'R03/current/')) . ?v33 skos:notation ?R03notation .}
-   optional {    ?dt ?rel4 ?v4 . filter(regex(str(?v4),'P09/current/')) . ?v4 skos:notation ?P09notation .}
-     optional {    ?dt ?rel5 ?v5 . filter(regex(str(?v5),'P02/current/')) . ?v5 skos:notation ?P02notation .}
+EOV_LIST = [
```
Comment (marc-portier) [inline src/marine_eov_broker/NVSQueries.py:8]: remember we are still waiting on the feedback if the full list of generated sparql is acutally useful / correct

File: src/marine_eov_broker/j2_templates/eov_query.sparql [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+  (EXISTS {?eov puv:chemicalObject ?eov_chemical . ?p01_mem ?rel3 ?eov_chemical .}) 
+  )
+  
+  # OUTPUT fetch
+  # for the remaining p01_members...
+  ?p01_mem skos:prefLabel ?p01lbl .FILTER(langMatches(lang(?p01lbl), "en")) .  # fetch the (english) label
+  ?p01_mem skos:notation ?P01not .                                             # fetch the code / id
+
+  optional {                                                                   # fetch if exists
+  	?p01_mem ?rel_r03 ?r03_mem .                                               #    a relation to
```
Comment (marc-portier) [inline src/marine_eov_broker/j2_templates/eov_query.sparql]: tabs versus spaces on lines 56-58 to fix indent

**Review by marc-portier — APPROVED** [review]
minor comments - non prohibiting a merge

File: src/marine_eov_broker/MarineRiBroker.py (hunk 2 above)
Comment (JotaFan) [reply src/marine_eov_broker/MarineRiBroker.py:461]: Agree! (for both comments)

**Review by marc-portier — APPROVED** [review]

File: src/marine_eov_broker/MarineRiBroker.py (hunk 3 above)
Comment (JotaFan) [reply src/marine_eov_broker/MarineRiBroker.py:440]: Added now.
Not sure if ipynb cleaning is needed though.
The best way to implement `submit_sparql_named_query` would also imply a refactor in most of the function the `submit_query` uses

Comment (marc-portier) [reply src/marine_eov_broker/MarineRiBroker.py:440]: yeah, I rest my case, goes outside the scope of what we aim for now

**Comment by JotaFan** [comment]
@galviset The packages are all up and running, feel free to merge this one after review

**Pull request merged**